docker-compose.yml

```
@@ -0,0 +1,12 @@
+version: "3.3"
+services:
+  app:
+    image: todo-app:dev
```

This currently relies on you running the correct `docker build ...` command to produce the correct `todo-app:dev` image.

You can instead do this to get docker-compose to handle the building
```suggestion
    build: 
      context: .
      target: development
```

Use `docker-compose up --build` to force it to check if the image needs rebuilding.